src/PublicApiGenerator/AttributeNameBuilder.cs

```diff
@@ -0,0 +1,10 @@
+namespace PublicApiGenerator
+{
+    public static class AttributeNameBuilder
+    {
+        public static string Get(string name)
+        {
+            return name != "System.ParamArrayAttribute" ? $"{name}{CodeNormalizer.AttributeMarker}" : name;
```

Comment:
```suggestion
            return name == "System.ParamArrayAttribute" ? name : $"{name}{CodeNormalizer.AttributeMarker}";
```

Reply:
I prefer my variant but I'm fine either way. Why do you prefer your suggestion?

Reply:
I assume that code is easier to read when there are fewer negative conditions. Then it is not necessary to mentally invert the condition.